src/RestrictedPython/transformer.py

```diff
@@ -1309,37 +1177,45 @@ def visit_While(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        self.not_allowed(node)
 
     def visit_Break(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        return self.node_contents_visit(node)
 
     def visit_Continue(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        self.not_allowed(node)
 
-#    def visit_Try(self, node):
-#        """
-#
-#        """
-#        return self.generic_visit(node)
+    def visit_Try(self, node):
+        """Allow Try-Finally without restrictions.
```

Comment:
This Comment was in order with visit_TryFinally, please review and move if necessary